### PR TITLE
LPS-184638  Add a checkbox to set/unset the propagation of permissions when creating a new folder

### PR DESCRIPTION
**Pull request by tototrinh**
Hi team,
Please help review this PR and leave your comments.
cc: @hhtank @locpham97 @ces-quanhuynh @ces-luannguyen
Origin PR: https://github.com/liferay-lima/liferay-portal/pull/4651
# Motivation
[LPS-184638](https://liferay.atlassian.net/browse/LPS-184638) Add a checkbox to set/unset the propagation of permissions when creating a new folder.

LPS-184638

# Solution
We modified the `liferay-ui:input-permissions` attribute by adding two new attributes: `permissionPropagationCheckboxLabel` and `permissionPropagationCheckboxValue`.

Regards.

### CODE REVIEW

File: util-taglib/src/META-INF/liferay-ui.tld
Comment (markocikos) [inline util-taglib/src/META-INF/liferay-ui.tld:1310]: `Checkbox` in API name is not ideal. In future, we might want to implement this feature with different UI. Maybe something like:
- `permissionPropagationLabel`
- `permissionPropagationEnabled`

Does that reflect meaning of attributes?

**Review by markocikos — COMMENTED** [review]
@tototrinh please see comment inline

File: util-taglib/src/META-INF/liferay-ui.tld
Comment (tototrinh) [reply util-taglib/src/META-INF/liferay-ui.tld:1310]: Yes, those names are good. Thanks for your feedback. I will update the PR.

**Comment by tototrinh** [comment]
Rename the attribute names

**Comment by tototrinh** [comment]
ci:test:sf

File: util-taglib/src/META-INF/liferay-ui.tld
Comment (tototrinh) [reply util-taglib/src/META-INF/liferay-ui.tld]: I updated the code. Please take a look again. Thanks.

**Review by markocikos — APPROVED** [review]
@tototrinh LGTM! :sparkles:

**Comment by tototrinh** [comment]
Hi @markocikos, could I have some updates on this, please? Should we wait for the rest of the parts of the original PR, or should we forward this to Brianchan separately? Thanks.

**Comment by markocikos** [comment]
>Should we wait for the rest of the parts of the original PR, or should we forward this to Brianchan separately?

@tototrinh As you wish. The frontend infra part of this PR passed review. If you're happy with it, and Lima is, feel free to `ci : forward`. If CI doesn't pass, you may want to followup with manual check from Lima QA and force forward.

**Comment by dsanz** [comment]
Hey @tototrinh do you plan to forward this PR to Brian, or would you like to send a different one?

Thanks

**Comment by tototrinh** [comment]
Hi @dsanz,
Currently, this PR is being blocked by another related PR that is being discussed with Brianchan. So this PR is waiting until we get approval from BrianChan and then we can re-submit another PR. Please feel free to close this PR.
Thanks.

**Pull request closed without merging**
